### PR TITLE
homework7

### PR DESCRIPTION
**Pull request by DenisKishylia**

### CODE REVIEW

File: Triphouse-project/index.html [hunk 1]
```diff
@@ -27,7 +27,7 @@
                     </li>
```
Comment (BubentsovKirill) [inline Triphouse-project/index.html:27]: 2021-04-19_22h14_23
Page is broken on mobile devices.

File: Triphouse-project/index.html [hunk 2]
```diff
@@ -132,6 +161,14 @@ <h4><a href="#" class="footer-content_link">Support</a></h4>
...
                 </ul>
         </div>
+        <div class="footer__mobile_wrapper">
+            <img src="./ikons/logo_vector (1).svg" />
```
Comment (BubentsovKirill) [inline Triphouse-project/index.html:165]: Word `ikons` doesn't exist. 
Why that `logo_vector (1)` isn't 1 word?

File: Triphouse-project/index.html [hunk 3]
```diff
@@ -60,6 +60,35 @@ <h1>
...
+            <div class="header__search_mob">
+                <p class="header__title_mobile">Discover stays<br/>
+                    to live, work or just relax</p>
+            </div>
+            <div class="header__download_mob">
+                <a href="#">
+                    <img src="./img/google-play-badge.svg" alt="Google Play"/>
+                </a>
+                <a href="#">
+                    <img src="./img/Download_on_the_App_Store_Badge_US-UK_RGB_blk_092917 1.svg" alt="App Store"/>
```
Comment (BubentsovKirill) [inline Triphouse-project/index.html:89]: The name of the file shout be short and without spaces. Check all file and rename.

**Comment by DenisKishylia** [comment]
In the commit "Alteraton and updatin" remade the project, for a small improvement. Added a mobile version for the lower and middle levels, unfortunately nothing worked for the upper one.

File: Triphouse-project/index.html [hunk 4]
```diff
@@ -1,33 +1,35 @@
 <!DOCTYPE html>
```
Comment (BubentsovKirill) [inline Triphouse-project/index.html:1]: 2021-04-22_21h50_13
Wrap and add to wrapper max-width: 1140px

File: Triphouse-project/index.html [hunk 5]
```diff
@@ -1,33 +1,35 @@
...
 <html lang="en">
```
Comment (BubentsovKirill) [inline Triphouse-project/index.html:2]: 2021-04-22_21h48_26

White line on the top ?
Text should be aligned to left.
Padding before logo ?

File: Triphouse-project/index.html [hunk 6]
```diff
@@ -47,21 +49,48 @@ <h1>
...
+           </div>
+        </div>
+        <div class="header__bg_mob">
+            <div class="header__wrapper_mob">
+                <img src="./ikons/logo_mob.svg" alt="">
+
             </div>
-          </div>
+            <div>
+                <h1 class="header_txt_mob">
```
Comment (BubentsovKirill) [inline Triphouse-project/index.html:71]: 2021-04-22_21h48_39
Why does that visible ?

**Pull request merged**